custom-resources/cert-manager-issuer.tf

```diff
@@ -1,7 +1,11 @@
 locals {
-  dns_zones           = try(var.core_services.azure_dns_zones.dns_zones, [])
-  dns_resource_group  = try(var.core_services.azure_dns_zones.resource_group, "")
-  enable_cert_manager = length(local.dns_zones) > 0 && length(local.dns_resource_group) > 0
+  enable_azure_dns = length(var.core_services.azure_dns_zones) > 0
+  zone_split_ids   = local.enable_azure_dns ? [for zone in var.core_services.azure_dns_zones.dns_zone : split("/", zone)] : []
```

Comment:
I'm not seeing `zone_split_ids` used anywhere else.

Reply:
Ah yea, we were using it by had to set something else up.